static/app/routes.tsx

```diff
@@ -2007,7 +2007,9 @@ function buildRoutes() {
       />
       <Route
         path="profile/:projectId/:eventId/"
-        component={make(() => import('sentry/views/profiling/profilesProvider'))}
+        component={make(
+          () => import('sentry/domains/profiling/providers/profilesProvider')
+        )}
```

Comment:
A little odd that the route is now rendering from the providers but not sure if there's a better way.

Reply:
Yeah, we can probably address this by making an index route, but yes, I agree